### ISSUE TITLE
Move `showNeovimInstallHelp` to redux+react

### ISSUE THREAD

**Issue by extr0py** [extr0py/oni#313]
Right now, UI/index.tsx has a method that looks like this:

```
export function showNeovimInstallHelp(): void {
    const element = document.getElementById("overlay-ui")
    ReactDOM.render(<InstallHelp />, element)
}
```

This should be refactored to be driven by Redux + React

**Comment by extr0py** [comment]
It could be potentially moved to `RootComponent`, we just need to have some redux state + actions to control rendering it.

**Comment by keforbes** [comment]
Fixed by #360.
